pkg/network/pasta/pasta.go

```diff
@@ -0,0 +1,183 @@
+package pasta
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"os/exec"
+	"strconv"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/rootless-containers/rootlesskit/pkg/api"
+	"github.com/rootless-containers/rootlesskit/pkg/common"
+	"github.com/rootless-containers/rootlesskit/pkg/network"
+	"github.com/rootless-containers/rootlesskit/pkg/network/iputils"
+	"github.com/rootless-containers/rootlesskit/pkg/network/parentutils"
+)
+
+// NewParentDriver instantiates new parent driver.
+// Requires slirp4netns v0.4.0 or later.
```

Comment:
`pasta` ?  I am not sure if past depends on `slirp4netns`

Reply:
Will fix later

Reply:
`pasta` does not depend on `slirp4netns`, that's kind of the point.